web/oauth_test.go

```diff
@@ -77,28 +77,19 @@ func TestAuthorizeOAuthApp(t *testing.T) {
 
 	// Test auth code flow
 	ruri, resp := ApiClient.AuthorizeOAuthApp(authRequest)
-	CheckNoError(t, resp)
+	require.Nil(t, resp.Error)
 
-	if len(ruri) == 0 {
-		t.Fatal("redirect url should be set")
-	}
+	require.NotZero(t, len(ruri), "redirect url should be set")
```

Comment:
Probably is more sematic here to use `require.NotEmpty` instead of `NotZero` + `len(ruri)`